src/views/Pipeline.vue

```diff
@@ -311,6 +318,18 @@ export default defineComponent({
         this.store.dispatch('job/fetchJobHistory', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => { event.target.complete() });
       }
     },
+    async reFetchJobs() {
```

Comment:
Use same method we use for pull to refresh, do not pass event and handle case

Reply:
Done Sir